Formula/grin-wallet.rb

```diff
@@ -0,0 +1,17 @@
+class GrinWallet < Formula
+  desc "Grin Wallet"
```

Comment:
Could you add more info about what the app does beyond just the name?

Reply:
I've added the following:
```
"Grin Wallet, official wallet for the cryptocurrency Grin"
```